bigquery/schema.go

```diff
@@ -346,6 +350,9 @@ func inferFieldSchema(fieldName string, rt reflect.Type, nullable bool) (*FieldS
 	case typeOfDateTime:
 		return &FieldSchema{Required: true, Type: DateTimeFieldType}, nil
 	case typeOfRat:
+		// TODO: big.Rat has arbitrary precision, so we can't discrimate without
```

Comment:
Should this be an open issue? Also, typo with discrimate.

Reply:
Updated comments